### DISCUSSION TITLE
allvariables can not be used

### DISCUSSION THREAD

**Discussion by Geriro36** [yalmip/YALMIP#974] [Problems using YALMIP] [answered]
Hi Johan,

My model encountered an unbounded problem. When I tried to debug according to _https://yalmip.github.io/debuggingunbounded_ . 

Undefined functions or variables **allvariables** are displayed.
This is still the case after I download the latest version of yalmip.

Why is that?

**Comment by johanlofberg** [comment, marked as answer]
It's still only in the develop branch, so you have to switch to that, or use the old approach described on the comments on the allvariables page
